internal/characters/klee/skill.go

```diff
@@ -9,10 +9,18 @@ import (
 
 var skillFrames []int
 
-const skillStart = 67
+var bounceHitmarks = []int{71, 111, 140}
+const mineHitmark = 240
 
 func init() {
-	skillFrames = frames.InitAbilSlice(67)
+	skillFrames = frames.InitAbilSlice(75)
+	skillFrames[action.ActionAttack] = 66
+	skillFrames[action.ActionCharge] = 69
+	skillFrames[action.ActionSkill] = 68
+	skillFrames[action.ActionBurst] = 34 // TODO: program early cancel for skill trigger
```

Comment:
I haven't added the E -> Q cancel that triggers a skill use without triggering the CD (useful for CW). Do we know if this is still relevant? I heard this was patched recently.

Reply:
It is still relevant, the only thing that got fixed is gaining multiple stacks by pressing E in quick succession.

Reply:
that part wasn't patched, you can cancel a phantom E and get a stack of cw/kagura.

the bug that was patched was double tapping E and getting a 2nd stack with 1 E usage